app/models/movie.rb

```diff
@@ -1,3 +1,13 @@
 class Movie < ApplicationRecord
   belongs_to :studio
+  has_many :movie_actors
+  has_many :actors, through: :movie_actors
+
+  def actors_list
+    actors.order(age: :asc)
+  end
+
+  def ave_actors_age
+    actors.average(:age).ceil
+  end
```

Comment:
I just want to make this quick note here that depending on who you ask, doing something like a `ceil` here could be considered presentation logic. I won't mark this as an MVC infraction, but something I wanted to bring up because some rubyist would consider this an infraction because it is technically presentation logic, I am in that camp.